packages/core/package.json

```diff
@@ -9,33 +9,17 @@
   "main": "dist/core.js",
   "dependencies": {
     "@nuxt/common": "^2.2.0",
+    "@nuxt/server": "^2.2.0",
```

Comment:
In the light of all the split packages, should core really have a dependency on server? If you use express or another server from create-nuxt-app you shouldn't need to install the default nuxt server as well I guess